### DISCUSSION TITLE
No need for the new feed

### DISCUSSION THREAD

**Discussion by rdw-software** [community/community#66263] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Here's what I want to see:

* A project I follow (watch, not starred) has new commits/releases
* Someone interacts with one of my repositories (watch, follow, open PR, discussion)
  * Edit: Actually, only the first two matter here since there's notifications for the rest

Here's what I *don't* want to see:

* Literally everything else, especially not
  * Algorithmically-generated content
  * Advertisements
  * Spam

Here's what I'm currently seeing (with the "new" feed):

* Two (yes, two) gigantic boxes, which take up the entire screen - that's way too large to be useful and a poor use of screen space
* Both are for release notes to some projects I've starred (NOT watched), years ago, and never interacted with

If I bother to scroll down, I'm seeing the following:

* "Trending" repositories that are 100% Chinese (unfortunately, I don't understand a word of Chinese)
* "Recommended for you": A documentation website for a project I clicked on once (dystopian/creepy to say the least)
* Five more releases for projects I've starred (not watched!) years ago, and of course have no interest in following
* Finally, ONE actual issue from a project I follow (watch), but it's 50% larger than before (literally, 210px vs 140px)
* And another release, from one of my own projects - why do I need to be notified about that? One can only wonder...

**Comment by AndyObtiva** [comment]
I agree. The old feed should have been retained AS IS via an option that should have been selected by default for all users until they change their new feed settings. That's Software Product Design 101! And, GitHub did follow that idea when they brought us the recommendation feed as a new tab a while ago. It's a shame they broke away from that software product design principle with this new feed. If they hadn't, none of the big issues encountered by everyone would have happened.
